docs/reference/search/multi-search.asciidoc

```diff
@@ -50,9 +53,16 @@ this endpoint the `Content-Type` header should be set to `application/x-ndjson`.
 [[search-multi-search-api-path-params]]
 ==== {api-path-parms-title}
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
+`<target>`::
+(Optional, string)
+Comma-separated list or wildcard (`*`) expression of the default data streams,
+indices, and index aliases used to limit the request.
```

Comment:
Nit: it took me a couple reads to understand that the request that was limited by the "default" target was the aggregate of all the individual searches.

Reply:
Thanks for the feedback. I updated the descriptions to clarify.